src/main/python/docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
License

Reply:
Well this makefile was create by the `sphinx-quickstart` command, so I do not know how copyright will be handled here.